src/main/java/com/bytelegend/Challenge.java

```diff
@@ -1,14 +1,11 @@
 package com.bytelegend;
 /**
- * 本类中的注释和Javadoc存在一些问题，会使得`mvn verify`失败。请修复之，确保`mvn verify`通过。
- * 注意：请只修复错误，不要修改注释内容。
- *
  * There are some issues in the comments and Javadocs in this class, which fails `mvn verify`.
  * Please fix it and make sure `mvn verify` pass.
- * Note that you should only fix the errors, don't modify the comments themselves. */
+ * Note that you should only fix the errors, don't modify the comments themselves.
  *
- * Location: C:\users\Documents
- * Fix the maximum number in three numbers, e.g. max(1,2,3) -> 3
+ * Location: C:\\users\\Documents
```

Comment:
Slashes in javadoc are different from they're in string literals. To display a slash in generated javadoc, you need `@literal` or HTML escape.